README.md

```diff
@@ -23,6 +23,95 @@ API documentation can be found [here](https://api.sia.tech/renterd).<br>
+
+---
+**NOTE**
+
+The name of these databases are configurable (see `RENTERD_DB_NAME` and
+`RENTERD_DB_METRICS_NAME`), so make sure to use the configured values, the
+following section will assume `renterd` defaults which are `renterd` and
+`renterd_metrics` for the main and metrics database respectively.
+
+---
+
+Depending on the user's configuration, these database are either SQLite or MySQL
+database. The following section outlines how to backup `renterd` in both
+scenarios.
+
+#### SQLite
+
+Backing up a SQLite database can be done using the following command:
+
+```bash
+sqlite3 db.sqlite ".backup 'db.bkp'"
+sqlite3 metrics.sqlite ".backup 'metrics.bkp'"
+```
+
+There is an alternative `.dump` command, which exports the database into a text
+file containing SQL statements. This backup is useful in its own, because it's
+more portable and can be used to import the contents of the database into
+another database on another system entirely. The `.backup` however yields a
+byte-for-byte replica of the original database file, it is usually a lot faster
+on large databases and it can be performed on a database that's actively being
+read or written to, even though we advise to shut down the renter before taking
+a backup.
+
+Restoring from a backup is as simple as putting the backup in place of the original.
+Another useful tool for backing up SQLite database is https://litestream.io/.
+
+#### MySQL
+
+Backuping up a MySQL database can be done using the `mysqldump` command. It's a
+utility provided by MySQL to backup or transfer a MySQL database. It's usually
+installed alongside the MySQL cient tools.
+
+The following command assumes MySQL is being ran from within a docker container:
+
+```bash
+docker exec [MYSQL_CONTAINER_NAME] /usr/bin/mysqldump -u [RENTERD_DB_USER] --password=[RENTERD_DB_PASSWORD] renterd > renterd_bkp.sql
+
+docker exec [MYSQL_CONTAINER_NAME] /usr/bin/mysqldump -u [RENTERD_DB_USER] --password=[RENTERD_DB_PASSWORD] renterd_metrics > renterd_metrics_bkp.sql
+```
+
+Restoring from this backup can be done using:
+
+```bash
+cat renterd_bkp.sql | docker exec -i [MYSQL_CONTAINER_NAME] /usr/bin/mysql -u [RENTERD_DB_USER] --password=[RENTERD_DB_PASSWORD] renterd
+
+cat renterd_metrics_bkp.sql | docker exec -i [MYSQL_CONTAINER_NAME] /usr/bin/mysql -u [RENTERD_DB_USER] --password=[RENTERD_DB_PASSWORD] renterd_metrics
+```
+
+### Partial Slabs
+
+For users that have upload packing enabled, it is very important to back up
+partial slabs alongside the database backups. These partial slabs are
+essentially a sort of buffer that gets uploaded to the network when that buffer
+reaches the size of a full slab, drastically speeding up a bunch of small file
+uploads. To ensure consistency between the database and these files on disk, it
+is recommended to gracefully shut the renter down before taking a backup of its
+database. These partial slabs are located in a folder called `partial_slabs`,
+right in the root folder.
+
```

Comment:
This is a good start but I think it can be streamlined more.

1. Instead of structuring it "SQLite", "MySQL" and then "Partial Slabs" I'd structure it in the way that you actually perform the backup. e.g. "Step1: Shutdown renter", "Step2: Create backup of database", "Step3: Copy partial slabs". 
2. Assume everything is running locally. Someone using docker will know how to adjust the commands
3. Shutting down the renter is very important for consistency so I'd make that a requirement
4. I wouldn't bring up litestream without a solution to the issue with consistency regarding partial slabs
5. Overall I'd stick to the backup process and avoid drifting off explaining concepts like partial slabs and alternatives like `.dump`. It should be good enough to mention that they are a folder worth backing up.
6. Restoring is more complicated than "putting the backup in place of the original". You have to assume that there is nothing left and a user starts on a new computer because the other one went up in flames.